qb/test/update.test.ts

```diff
@@ -35,7 +35,7 @@ test("update", async () => {
   e.update(e.Bag, () => ({
     set: {
       stringsMulti: {
-        "+=": "new string",
+        "+=": ["new string"] as ["new string"],
```

Comment:
I think we should allow non array `string`s here as well since it's modifying the existing multi set.